package.json

```diff
@@ -89,6 +89,7 @@
     "@scure/bip32": "^1.3.0",
```

Comment:
We'll address this in another PR when live, but this should either be more generic or explicitly call out other supported networks other than Ethereum.

<img width="329" alt="Good Til Time" src="https://github.com/user-attachments/assets/e23a27de-9508-44e3-a811-5fce1f24c5b4">